package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "lint": "eslint packages tests --cache --fix --ext=js --ext=ts",
-    "pretest": "npm run lint --silent && npm run typecheck",
```

Comment:
why that? we should make sure that the code is linted

Reply:
@pozylon typescript fails on locally that why i remove it and forgot, putting it back.

Reply:
@Mikearaya ping